### PR TITLE
feat: add '--store-key-prefix' to support multiple meta instances can use the same etcd backend

### PR DESCRIPTION
**Pull request by zyy17**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

When deploying multiple clusters, it's more convenient and cost-effective to re-use the same etcd cluster.

In this PR, I added `--store-key-prefix` which is specified by the admin in the metasrv to support that multiple metasrv instances can use the same etcd cluster without key confliction.

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

**Review by fengjiachun — APPROVED** [review]
LGTM

File: src/meta-srv/src/bootstrap.rs [hunk 1]
```diff
@@ -189,9 +190,24 @@ pub async fn metasrv_builder(
...
+                kv_backend,
+                Some(
+                    EtcdElection::with_etcd_client(
+                        &opts.server_addr,
+                        etcd_client.clone(),
+                        opts.store_key_prefix.clone(),
+                    )
+                    .await?,
+                ),
                 Some(EtcdLock::with_etcd_client(etcd_client)?),
```
Comment (fengys1996) [inline src/meta-srv/src/bootstrap.rs:211]: It seems that the name of etcd lock should also be prefixed with the name to distinguish it.

**Review by fengys1996 — APPROVED** [review]
Rest LGTM

File: src/meta-srv/src/bootstrap.rs (hunk 1 above)
Comment (zyy17) [reply src/meta-srv/src/bootstrap.rs:211]: You're right. I will fix it.

Comment (zyy17) [reply src/meta-srv/src/bootstrap.rs]: @fengys1996 already add the prefix for the lock.

**Comment by killme2008** [comment]
Question: it this parameter only be passed by the command line arg? Can the users configure it in `metasrv.toml`? @zyy17 @fengjiachun

**Comment by fengjiachun** [comment]
Oh, it seems that it cannot be configured through the toml file, I think we need a way with toml. @zyy17

**Pull request merged**